crates/config/src/config.rs

```diff
@@ -23,8 +23,7 @@ pub struct Config {
     // TODO(onbjerg): Can we make this easier to maintain when we add/remove stages?
     pub stages: StageConfig,
     /// Configuration for pruning.
-    #[cfg_attr(feature = "serde", serde(skip_serializing_if = "Option::is_none"))]
-    pub prune: Option<PruneConfig>,
+    pub prune: PruneConfig,
```

Comment:
this is better but should this be marked with serde(default) because this is now mandatory and previously it was not